lib/common/logging.c

```diff
@@ -334,6 +334,11 @@ crm_add_logfile(const char *filename)
     if(is_default) {
         default_fd = fd;
 
+        // Some resource agents will log only if environment variable is set
+        if (daemon_option("logfile") == NULL) {
+            set_daemon_option("logfile", filename);
+        }
+
```

Comment:
Just a sidenote, when `none` requested in particular for `PCMK_logfile`,
it won't even get here.

Likely a desired behaviour, just stating the obvious.